src/Common/src/TypeSystem/Interop/MethodDesc.Interop.cs

```diff
@@ -28,6 +28,40 @@ public virtual PInvokeMetadata GetPInvokeMethodMetadata()
         {
             return default(PInvokeMetadata);
         }
+
+        /// <summary>
+        /// If <see cref="IsPInvoke"/> is true, retrieves the parameters of the platform invoke.
+        /// </summary>
+        public virtual PInvokeArgumentMetadata[] GetPInvokeArgumentMetadata()
+        {
+            return default(PInvokeArgumentMetadata[]);
+        }
+    }
+
+    [Flags]
+    public enum PInvokeArgumentAttributes
+    {
+        None = 0,
+        In = 1,
+        Out = 2,
+        Lcid = 4,
```

Comment:
System.Reflection.ParameterAttributes does define them for whatever reason. These might be obsolete values in early versions of runtime that are never used (the 'holes' are probably a good indication).